### PR TITLE
Fix: Loading Field group in Query component

### PR DESCRIPTION
**Pull request by tumms2021389**
Fixing issue loading data reference inside Field group (Query)

### CODE REVIEW

File: src/components/templates/FieldGroupTemplate/index.tsx [hunk 1]
```diff
@@ -54,7 +53,7 @@ export default function FieldGroupTemplate(props) {
   pConn.setInheritedProp('displayMode', 'LABELS_LEFT');
   const memoisedReadOnlyList = useMemo(() => {
     return referenceList.map((item, index) => (
-      <FieldGroup item={item} key={index} name={`${HEADING} ${index + 1}`} />
+      <FieldGroup item={item} key={item.ProductName} name={`${HEADING} ${index + 1}`} />
```
Comment (tumms2021389) [inline src/components/templates/FieldGroupTemplate/index.tsx:56]: We shouldn't hardcode productName here as that might get changed for other data objects.

Comment (JEAxero) [reply src/components/templates/FieldGroupTemplate/index.tsx:56]: Ok. So there's nothing we can readily do for now other than use index and turn off the warning?

Comment (tumms2021389) [reply src/components/templates/FieldGroupTemplate/index.tsx]: yes, as we are using the index as a key in the read-only element, it mostly won't create any problem for now

**Review by JEAxero — APPROVED** [review]
Would like to, eventually, come up with some better "keys" to eliminate the console warning that can show up. But we can leave that for another day. For now, tested and looks ok.

**Pull request merged**